### ISSUE TITLE
Hyper-V: SSH after Password change

### ISSUE THREAD

**Issue by wahmedswl** [rancher/os#1773]
**RancherOS Version: (ros os version)**
v0.9.2
**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
Hyper-V

I created a VM in Hyper-V and after login to Rancher in the Hyper-V, i changed password via ```sudo passwd``` and after that tried to ssh to the VM to install RancherOS on the disk as Hyper-V console is really not that good but i cannot ssh into the VM. I had this working as this was the common workflow i follow to create VMs but it no longer work with v0.9.2. Please see #1467

After that, ```cloud-config.yml``` is created and install to disk is done via ```sudo ros install -c cloud-config.yml -d /dev/sda --append "rancher.password=password"``` but i cannot do that now cuz i am stuck at the ssh step.

Any help would be appreciated, Thanks

**Comment by SvenDowideit** [comment]
@wahmedswl did it work in 0.9.0? I guess I need to upgrade my windows box from Windows 10 Home :(

**Comment by wahmedswl** [comment]
@SvenDowideit nope, i tried v0.9.0 and i cannot ssh on that also. I am downloading v0.8.2 and will let u know after checking it.

Thanks

**Comment by wahmedswl** [comment]
Hi, i tried multiple RancherOS versions and it seems like some sort of issue with my Machine. I even tried running v0.8.0-rc2 which i tested earlier and i cannot ssh into that also.

Might be i am forgetting some step or something else changed in my Machine, i have another Ubuntu box which i ssh frequently and its working fine now. Will update if i get something or if anything you guys can help in identifying the issue.

Any help would be appreciated.

Thanks

**Comment by SvenDowideit** [comment]
I'm just finishing up releasing https://github.com/rancher/os/releases/tag/v1.0.0 - so I don't have things in a place where I can help out yet. 

While you can't ssh in, you can still use the hyper-v console? I'm wondering if you can get out from there (and what kind of network do you have between the vm and your local shell?)

are you getting a connectivity error from ssh, or an authentication issue?

**Comment by wahmedswl** [comment]
@SvenDowideit i am getting authentication issue
```
rancher@192.168.7.79's password:
Permission denied (publickey,password,keyboard-interactive).
```

**Comment by SvenDowideit** [comment]
mmm, what console are you using? the default, or did you switch?

**Comment by wahmedswl** [comment]
@SvenDowideit  nope, i didn't change anything. Fresh VM bootup and direct password changed. It seems like issue is with the machine as i cannot ssh into CoreOS VM i created in HyperV after changing its password.

I had same workflow in the past and it seems to be stopped working now, might be Windows update as i am on Insider Build or some configuration issue.
